### PR TITLE
User can view items for sale.

### PR DESCRIPTION
**Pull request by stephyacek**
screen shot 2017-06-14 at 2 32 24 pm

screen shot 2017-06-14 at 2 33 39 pm

screen shot 2017-06-14 at 2 32 34 pm

### CODE REVIEW

**Review by ronperris — CHANGES_REQUESTED** [review]
Please move the art into an array in JavaScript and draw the list dynamically.

File: main.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-unused-vars */
+
+var product = [
```
Comment (ronperris) [inline main.js]: I think this should be products, plural.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  { image: 'images/photo1.jpg',
```
Comment (ronperris) [inline main.js]: These names should either be descriptions of the images or have a hyphen between image and the number.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    title: 'Something With Turquoise'
+  },
+  { image: 'images/photo2.jpg',
```
Comment (ronperris) [inline main.js]: The starting { should be on a separate line from the image property.

File: main.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  },
+  { image: 'images/photo7.jpg',
+    title: 'Zealous Eater'
+  },
+  { image: 'images/photo8.jpg',
+    title: 'Flaming Fuschia'
+  }
+]
+
+var $artworkSection = document.querySelector('.artwork')
```
Comment (ronperris) [inline main.js:38]: Could this just be called $artwork?

File: main.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+document.addEventListener('DOMContentLoaded', renderPhotos(product))
```
Comment (ronperris) [inline main.js]: You are calling renderPhotos here, did you actually want to pass it to the event listener instead?

File: main.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+function createEachProduct(product) {
+  var $span = document.createElement('span')
```
Comment (ronperris) [inline main.js]: Would $product be a better name? Then you don't need it in all the other variable's names.

File: main.js [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  $productDescription.setAttribute('class', 'product-description')
+  $image.setAttribute('src', product.image)
+  $span.setAttribute('class', 'product')
+  $span.appendChild($image)
+  $span.appendChild($productDescription)
+  $productDescription.textContent = product.title
+  return $span
+}
+
+function renderPhotos(product) {
```
Comment (ronperris) [inline main.js]: This parameter should be products, it is a collection of more than on product.

File: main.js [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+  for (var i = 0; i < product.length; i++) {
+    var eachProduct = createEachProduct(product[i])
```
Comment (ronperris) [inline main.js]: This should be $product, it is an element.

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+header {
+  font-family: 'Fredericka the Great', sans-serif;
+  text-align: center;
+  font-size: 4rem;
+  padding: 3rem;
+}
+
+.title-cat {
+  max-width: 35px;
```
Comment (ronperris) [inline styles.css]: Could this be in rem? Like the rest of the values your using for spacing.

File: main.js (hunk 2 above)
Comment (stephyacek) [reply main.js]: So renaming my saved images "photo-1", "photo-2" etc?

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+  display: inline-block;
+  text-align: center;
+  border: 1px solid black;
+  margin: 40px;
+}
+
+footer {
+  background-color: #acb6bb;
+  color: white;
+  font-weight: lighter;
```
Comment (ronperris) [inline styles.css]: This is using lighter and above your using 600. Make them consistent.

**Review by ronperris — CHANGES_REQUESTED** [review]
Please fix the issues, check the inline comments for details.

**Review by ronperris — APPROVED** [review]

**Review by ronperris — APPROVED** [review]
Looks good to merge.

**Pull request merged**